### DISCUSSION TITLE
关于Absolute软件的使用和数据处理

### DISCUSSION THREAD

**Discussion by XiaobenLi00** [ShixiangWang/self-study#77] [Q&A] [answered]
版主好，我正在进行我的本科毕业设计，是跟生物信息相关的，但是我本科学的是计算机所以对生物信息不是很了解。现在我需要分析肿瘤数据的克隆性以及LOH突变，目前在染色体数据中植入一些突变后使用ART模拟测序软件生成测序数据，但是这些数据无法直接作为absolute的输入，想问一下这些原始数据应该如何处理成segment文件和maf文件使得absolute能够处理呢，感谢版主，希望能得到答复！

**Comment by ShixiangWang** [comment, marked as answer]
从测序数据到拷贝数数据你可以用 CNV 和 mutation 检测软件，网上应该很多。
